src/Bridges/intervalbridge.jl

```diff
@@ -48,6 +48,30 @@ function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintDual, c::SplitIntervalBr
     return lower_dual > -upper_dual ? lower_dual : upper_dual
 end
 
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintBasisStatus, c::SplitIntervalBridge)
+    lower_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.lower)
+    upper_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.upper)
+    if lower_stat == MOI.NONBASIC_AT_LOWER
+        Compat.@warn("Single sided constraints should not have basis status:" *
+            " NONBASIC_AT_LOWER, instead use NONBASIC.")
+    end
+    if upper_stat == MOI.NONBASIC_AT_UPPER
+        Compat.@warn("Single sided constraints should not have basis status:" *
+            " NONBASIC_AT_UPPER, instead use NONBASIC.")
+    end
+    if lower_stat == MOI.NONBASIC
+        return MOI.NONBASIC_AT_LOWER
+    end
+    if upper_stat == MOI.NONBASIC
+        return MOI.NONBASIC_AT_UPPER
+    end
+    if lower_stat != upper_stat
+        Compat.@warn("Basis status of lower ($lower_stat) and upper ($upper_stat) constraint are inconsistent," *
```

Comment:
Use backsticks around lower_stat and upper_stat since it's Julia constant, not english words